cmake/README.md

```diff
@@ -40,29 +36,46 @@ cmake <options> <repo_root>
 ```
 
 Some useful options:
-- `-G <generator>` - specify particular project generator, see `cmake --help` for details.
-- `-DCMAKE_BUILD_TYPE=Debug` - specify for Debug build, it isn't applicable for multi-config generators, e.g. for Visual Studio generator).
+- `-G <generator>` - specify particular project generator. See `cmake --help` for details.
+- `-DCMAKE_BUILD_TYPE=Debug` - specify for Debug build. It is not applicable for multiconfig generators, e.g. for Visual Studio* generator.
 
 #### TBBBind library configuration
 
-The TBBbind library has three versions: `tbbbind`, `tbbbind_2_0` and `tbbbind_2_5`. Each of these versions is linked with corresponding HWLOC library version: `tbbbind` links with HWLOC 1.11.x , `tbbbind_2_0` links with HWLOC 2.1-2.4, `tbbbind_2_5` links with HWLOC 2.5 and later.
+The TBBbind library has three versions: `tbbbind`, `tbbbind_2_0`, and `tbbbind_2_5`. Each of these versions is linked with corresponding HWLOC library version: 
+- `tbbbind` links with HWLOC 1.11.x 
+- `tbbbind_2_0` links with HWLOC 2.1–2.4
+- `tbbbind_2_5` links with HWLOC 2.5 and later
+
+The search for a suitable version of the HWLOC library is enabled by default. If you want to use a specific version of the library, you can specify the path to it manually using the following CMake variables:
 
-The search for a suitable version of the HWLOC library is enabled by default, but if you want to use a specific version of the library, you can specify the path to it manually using the following CMake variables:
+ - `CMAKE_HWLOC_<HWLOC_VER>_LIBRARY_PATH` - path to the corresponding HWLOC version shared library on Linux* OS or path to `.lib` file on Windows* OS
+ - `CMAKE_HWLOC_<HWLOC_VER>_INCLUDE_PATH` - path to the corresponding HWLOC version including directory
 
- - `CMAKE_HWLOC_<HWLOC_VER>_LIBRARY_PATH` - path to the corresponding HWLOC version shared library on Linux or path to `.lib` file on Windows.
- - `CMAKE_HWLOC_<HWLOC_VER>_INCLUDE_PATH` - path to the corresponding HWLOC version includes directory.
 
-Automatic HWLOC searching requires CMake >= 3.6.
+---
+**NOTE:** Automatic HWLOC searching requires CMake version 3.6 or higher.
 
-Windows requires one additional variable for correct TBBBind library building:
+---
+
+
+Windows* OS requires an additional variable for correct TBBBind library building:
  - `CMAKE_HWLOC_<HWLOC_VER>_DLL_PATH` - path to the corresponding HWLOC version `.dll` file.
 
-`HWLOC_VER` substring used above can be replaced with one of the three values: `1_11` for the `tbbbind` library configuration, `2` for the `tbbbind_2_0`, and `2_5` for the `tbbbind_2_5` library configuration.
+`HWLOC_VER` substring used earlier can be replaced with one of the three values:
+- `1_11` for the `tbbbind` library configuration
+- `2` for the `tbbbind_2_0`
+- `2_5` for the `tbbbind_2_5` library configuration
```

Comment:
Please align: either remove `library configuration` or add it after `tbbbind_2_0`.

Reply:
will do